### PR TITLE
Add graphql query and schema support

### PR DESCRIPTION
**Pull request by vektah**
Pretty basic support, there are probably a heap of edge cases, but it should be a good start.

[Spec](https://facebook.github.io/graphql/)

### CODE REVIEW

File: lexers/g/graphql.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	&Config{
+		Name:      "GraphQL",
+		Aliases:   []string{"graphql", "graphqls", "gql"},
+		Filenames: []string{"*.graphql", "*.graphqls"},
+	},
+	Rules{
+		"root": {
+			{`(query|mutation|subscription)\b`, KeywordDeclaration, nil},
+			{`[^\W\d]\w*`, NameClass, nil},
+			{`[^\W\d]\w*`, NameProperty, nil},
```
Comment (alecthomas) [inline lexers/g/graphql.go]: I don't think this or the next will ever match.

File: lexers/lexers_test.go [hunk 2]
```diff
@@ -64,15 +70,21 @@ func TestLexers(t *testing.T) {
 			actual, err := chroma.Tokenise(lexer, nil, string(actualText))
 			assert.NoError(t, err)
 
-			// Read expected JSON into token slice.
-			var expected []chroma.Token
-			r, err := os.Open(expectedFilename)
-			assert.NoError(t, err)
-			err = json.NewDecoder(r).Decode(&expected)
-			assert.NoError(t, err)
+			if *write {
```
Comment (vektah) [inline lexers/lexers_test.go]: Allows you to run `go test ./... -write` to write the current lexer output to the expected file. I left this in because I found it very useful for making changes, happy to take it out or put it in another PR, just let me know.

Comment (alecthomas) [reply lexers/lexers_test.go]: Awesome!

Can you update lexers/testdata/README.md with these new instructions?

Comment (vektah) [reply lexers/lexers_test.go]: Ha! I didn't even see the readme. That makes sense I'll delete this change.

Comment (alecthomas) [reply lexers/lexers_test.go]: Nooo! I prefer this approach! It just needs to be documented.

**Comment by alecthomas** [comment]
Ping.

File: lexers/lexers_test.go (hunk 2 above)
Comment (vektah) [reply lexers/lexers_test.go]: I'll push up a separate pr later this afternoon.
